.editorconfig

```diff
@@ -9,6 +9,7 @@ end_of_line = lf
 indent_size = 2
 indent_style = space
 insert_final_newline = true
+max_line_length = 100
```

Comment:
Is there any additional setup that we need to do to get this to work? All of the other rules seem to be working fine on my end other than this rule.

Reply:
I don't think so, though maybe I should move it to just be for Markdown. I would like this to be set for python but didn't set it in `black`'s settings because it would have led to a lot of noise in this pr. Is it not working for any files for you? Or do you mean it isn't working for python files?

Reply:
Yeah I saw the details of your commit msg indicating the reason for not including in the `black` settings. For me its not working in the markdown files for some reason. (I was trying to do a local diff since the current one on github is hard to parse and wasn't able to get it to work.)

Reply:
Weird, I wonder why. You use VS code right?

Reply:
Oh I see, apparently it isn't implemented in the VS code plugin because they don't think it should be part of it: https://github.com/editorconfig/editorconfig-vscode/issues/53

Reply:
Hmm I guess maybe I should remove it completely then, otherwise we'll keep having differences between people using vscode and people using jetbrains IDEs

Reply:
Yeah, I'm using VSCode. What's the rationale behind having max line lengths for md files? I've generally been a fan of using wrapping as it makes the diffs cleaner imo.

Reply:
Mainly being able to have multiple things up side by side. I had the plain README, the preview, and a code window all side by side to help as I was writing and with the original super long lines it was harder to edit. Wrapping sometimes works weirdly, but I can change to using other screens or something.